spec/support/command_helpers.rb

```diff
@@ -296,6 +296,40 @@ def set_date
     send_command_and_confirm @task, command_list, "intention to set date"
   end
 
+  def verify_synchronization
```

Comment:
if this code is not reused between several tests, it should be placed directly in the rpsec test

Reply:
Cool, I'll move it there.

Reply:
The reseting of time part is done in multiple places though, so I made a helper function called reset_Date which does just that.